Appraisals

```diff
@@ -1,22 +1,12 @@
 # frozen_string_literal: true
 
 case RUBY_VERSION
-when '3.1.3', '3.2.1', '3.3.0'
-  appraise "ruby-#{RUBY_VERSION}_actionview61" do
-    source 'https://rubygems.org' do
-      gem 'actionview', '~> 6.1.0'
-    end
-  end
-
-  appraise "ruby-#{RUBY_VERSION}_actionview70" do
-    source 'https://rubygems.org' do
-      gem 'actionview', '~> 7.0.0'
-    end
-  end
-
-  appraise "ruby-#{RUBY_VERSION}_actionview71" do
-    source 'https://rubygems.org' do
-      gem 'actionview', '~> 7.1.0'
+when '3.2.5', '3.3.6', '3.4.1'
+  ['7.0.0', '7.1.0', '7.2.0', '8.0.0'].each do |actionview_version|
+    appraise "ruby-#{RUBY_VERSION}__actionview#{actionview_version}" do
+      source 'https://rubygems.org' do
+        gem 'actionview', "~> #{actionview_version}"
+      end
```

Comment:
Not against doing it this way, but it can get confusing if there are increasing levels of appraisals.